app/classes/models.py

```diff
@@ -1,34 +0,0 @@
-from sqlalchemy import Column,Integer, String, ForeignKey, create_engine
-from sqlalchemy.ext.declarative import declarative_base
-from sqlalchemy.orm import sessionmaker
-
-Base = declarative_base()
-
-class PeopleModel(Base):
-	__tablename__ = 'people'
-	id = Column(Integer, primary_key = True, autoincrement = True)
```

Comment:
autoincrement is not necessary here because you have already set it to primary key.